raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,332 @@
+            - assert: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000, balance: 189_000_000_000_000_000, state: "opened"}
+            # 1 sends 10 tokens to 0
+            - transfer: {from: 1, to: 0, amount: 10_000_000_000_000_000, lock_timeout: 30}
+            - wait: 5
+            - assert: {from: 0, to: 1, total_deposit: 260_000_000_000_000_000, balance: 10_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000, balance: 299_000_000_000_000_000, state: "opened"}
+            # 4 sends 10 tokens to 2 (0 -> 1-> 2)
+            - transfer: {from: 4, to: 2, amount: 10_000_000_000_000_000, lock_timeout: 30}
+            - wait: 5
+      - parallel:
+          name: "Assert for the channel states after mediated transfers were completed"
+          tasks:
+            - assert: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000, balance: 0, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000, balance: 200_000_000_000_000_000, state: "opened"}
+            - assert: {from: 0, to: 1, total_deposit: 260_000_000_000_000_000, balance: 0, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000, balance: 309_000_000_000_000_000, state: "opened"}
+            - assert: {from: 4, to: 0, total_deposit: 25_000_000_000_000_000, balance: 15_000_000_000_000_000, state: "opened"}
+            - assert: {from: 0, to: 4, total_deposit: 10_000_000_000_000_000, balance: 20_000_000_000_000_000, state: "opened"}
+      - serial:
+          name: "4 Closes his channel with 0"
+          tasks:
+            - close_channel: {from: 4, to: 0}
+            - wait: 5
+      - parallel:
+          name: "Check if channel was closed"
+          tasks:
+            - assert: {from: 4, to: 0, total_deposit: 25_000_000_000_000_000, balance: 15_000_000_000_000_000, state: "closed"}
+            - assert: {from: 0, to: 4, total_deposit: 10_000_000_000_000_000, balance: 20_000_000_000_000_000, state: "closed"}
+      - parallel:
+          name: "All nodes try to make a payment to 4"
+          tasks:
+            - transfer: {from: 0, to: 4, amount: 1_000_000_000_000_000, expected_http_status: 409, lock_timeout: 30}
+            - transfer: {from: 1, to: 4, amount: 1_000_000_000_000_000, expected_http_status: 409, lock_timeout: 30}
+            - transfer: {from: 2, to: 4, amount: 1_000_000_000_000_000, expected_http_status: 409, lock_timeout: 30}
+            - transfer: {from: 3, to: 4, amount: 1_000_000_000_000_000, expected_http_status: 409, lock_timeout: 30}
+      - serial:
+          name: "0 and 1 leave token netwwork"
+          tasks:
+            - leave_network: {from: 0}
+            - leave_network: {from: 1}
+            - wait: 20
+      - serial:
+          name: "Check if the only channel open is 2 <-> 3"
```

Comment:
You don't check if it's the only channel open. You just check that it is open. Not that none of the others are not.